app/assets/stylesheets/puppetclasses.scss

```diff
@@ -2,5 +2,26 @@
   .fields {
     border-top: 1px dotted lightgrey;
     padding-top: 10px;
+    &:nth-child(3) {
+      border-top: none;
+    }
   }
 }
+
+.lookup-keys-container {
+  h6 {
+    margin-top: 0px;
+  }
+
+  .help-block {
+    margin: 0px;
+  }
+
+  input[type=checkbox]{
+    margin-top: 11px;
+  }
+}
+
+.btn-fullscreen{
```

Comment:
I think this should be in application as it affects the whole app.  You could perhaps restrict it to lookup-keys-container, but it seems to also be necessary on the Parameters tab of host#edit, though I can't tell why it's now necessary to make that look the same as it did.